gateway/src/apicast/policy/scope_check/scope_check.lua

```diff
@@ -0,0 +1,105 @@
+local policy = require('apicast.policy')
+local _M = policy.new('Scope Check Policy')
+
+local ipairs = ipairs
+local MappingRule = require('apicast.mapping_rule')
+
+local new = _M.new
+
+local function init_mapping_rules(scopes)
+  for _, scope in ipairs(scopes) do
+    scope.mapping_rule = MappingRule.from_proxy_rule({
+      http_method = 'GET',
```

Comment:
I think it would be better if this would be some non http method keyword like 'ANY'. Also it could be in a variable so it is reused when calling `scope.mapping_rule:matches`.

Reply:
Changed.